app.js

```diff
@@ -878,8 +878,7 @@ async function appHomeUnassignedIssues(userId, client) {
     } catch (error) {
       console.log(
         "Couldn't find user",
-        result.fields.reporter.emailAddress,
-        error,
```

Comment:
This was overly verbose, putting a lot of info into logs, if we do need to debug it more we can try add this but this is normal sometimes if a user doesn't exist anymore, so I don't want tons of log spam for just opening the app home